lib/utils/logger.js

```diff
@@ -11,38 +11,34 @@
 const debug = require('debug');
```

Comment:
Not too happy with changes in this class.

1) You are caching `this.logger` on `Logger` class (that may confusing as `this` is used in static method). Make a const declaration after class with is an instance of class as `logger = new Logger()`.

2) We can probably get rid of both old and new `getLogger` method, just export instance `logger` of `Logger` class created in step (1), require and use that everywhere